src/components/DashboardBody.js

```diff
@@ -12,7 +12,7 @@ const buildWidgets = widgets =>
       'Image': <Image { ...widget } key={ widget.title } />,
       'QRCode': <QRCode { ...widget } key={ widget.title } />,
       'Clock': <Clock { ...widget } key={ widget.title } />
-    }[ widget.type ]))
+    }[ widget.type ] ) )
```

Comment:
This looks funny as well, seems like we need another exception for square brackets.